docs/en/latest/plugins/forward-auth.md

```diff
@@ -117,32 +117,35 @@ We can perform the following three tests.
 1. **request_headers** Send Authorization header from `client` to `authorization` service
 
 ```shell
-$ curl http://127.0.0.1:9080/headers -H 'Authorization: 123'
-{
+curl http://127.0.0.1:9080/headers -H 'Authorization: 123' \
+-d '{
```

Comment:
Incorrent modification, this is the response.

Reply:
Good catch! I changed this to two text blocks. PTAL!